src/status_im/navigation/screens.cljs

```diff
@@ -949,7 +951,19 @@
     :metrics   {:track?   :true
                 :alias-id :keycard.pin.create}
     :options   {:insets {:top? true :bottom? true}}
-    :component pin.create/view}])
+    :component keycard.pin.create/view}
+
+   {:name      :screen/keycard.pin.enter
+    :metrics   {:track?   :true
+                :alias-id :keycard.pin.enter}
+    :options   {:insets {:top? true :bottom? true}}
+    :component keycard.pin.enter/view}
+
+   {:name      :screen/keycard.profile-keys
+    :metrics   {:track?   :true
+                :alias-id :keycard.profile-keys}
+    :options   {:insets {:top? true :bottom? true}}
+    :component keycard.migrate.profile-keys/view}])
```

Comment:
Thanks for enabling the tracking on these screens 🙌 

Also it seems we can remove `alias-id` for these entries, since internally we'll use the expression `(name (:name screen))` by default.